languages/go/xorm/run.sh

```diff
@@ -26,28 +26,9 @@ subproject_teardown() {
   docker compose down
 }
 
-subproject_examples() {
```

Comment:
Removed this command as no longer necessary (see comment in description). The tests are the source of truth for examples. No need to duplicate.